falcon/util/time.py

```diff
@@ -20,6 +23,13 @@ class TimezoneGMT(datetime.tzinfo):
 
     GMT_ZERO = datetime.timedelta(hours=0)
 
+    def __init__(self) -> None:
```

Comment:
Maybe we could just plop the `@deprecation.deprecated(...)` decorator on `__init__`, just like we do in `falcon.app.API`?